src/NAR/RunTime

```diff
@@ -0,0 +1,77 @@
+package NAR;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class RunTime {
+
+	public ArrayList<String> listOfLines(String str) {
+
+		ArrayList<String> lines = new ArrayList<>();
```

Comment:
While ArrayList is good you should store your ArrayList object in a List object which will make the code more flexible (if you need help doing that I can show you how)